pages/common/git-pull.md

```diff
@@ -0,0 +1,11 @@
+# git pull
+
+> Fetch branch from a remote repo and merge it to current repo
+
+- Download the remote-tracking branch from default remote and merge it
```

Comment:
I also don't like "remote-tracking branch". Grabbed it from man. What do you think is the best?